samples/FunctionApp/FunctionApp.csproj

```diff
@@ -13,8 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.13.0-preview5-20230303.1" />
```

Comment:
I think you still need a worker package reference here (latest is 1.10.1) or are you going to use a project reference?

Reply:
the project reference is below in the file - we don't have a preview package out for this feature yet so not sure what this one is referencing tbh.